dlib/dnn/tensor_abstract.h

```diff
@@ -467,7 +467,7 @@ namespace dlib
         !*/
 
         explicit resizable_tensor(
-            long long n_, long long g k_ = 1, long long nr_ = 1, long long nc_ = 1
+            long long n_, long long k_ = 1, long long nr_ = 1, long long nc_ = 1
```

Comment:
I guess regardless of what is done to #1148, at least this change should go to `master`. (Sorry, my bad...)